py/scheduler.c

```diff
@@ -28,17 +28,21 @@
 
 #include "py/runtime.h"
 
-#if MICROPY_KBD_EXCEPTION
-// This function may be called asynchronously at any time so only do the bare minimum.
-void MICROPY_WRAP_MP_KEYBOARD_INTERRUPT(mp_keyboard_interrupt)(void) {
-    MP_STATE_VM(mp_kbd_exception).traceback_data = NULL;
-    MP_STATE_VM(mp_pending_exception) = MP_OBJ_FROM_PTR(&MP_STATE_VM(mp_kbd_exception));
+void mp_sched_exception(mp_obj_t exc) {
```

Comment:
Is MICROPY_WRAP_MP_SCHED_EXCEPTION missing here?

Reply:
Indeed!  Thanks for that.  See #7199.